screens/ImagesScreen/index.js

```diff
@@ -30,7 +30,6 @@ const ImagesScreen = ({navigation}) => {
     }
     setModalVisible(false);
     setSheet('');
-    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [sheet]);
```

Comment:
you should fix linter issue, not turn off them ))

Reply:
Ok, I will fix it )